src/core/services/actions-handler/actions-handler-service.js

```diff
@@ -21,7 +21,13 @@ export default async function ActionsHandlerService(options) {
   const tokenError = 'loan token not found. please try again later.';
   const borrowError =
     'This book is not available to borrow at this time. Please try again later.';
-  const erroneousActions = ['browse_book', 'borrow_book', 'create_token'];
+  const erroneousActions = [
+    'browse_book',
+    'borrow_book',
+    'create_token',
+    'renew_loan',
+    'return_loan',
```

Comment:
@nsharma123 is this registration correct? will xhr errors be handled for these states automatically?

Reply:
Yes, there is error checkbox on demo which add a GET param in URL to check if we want to see an error on these actions.